trace-recorder/src/main/org/jetbrains/lincheck/trace/recorder/TraceCollectingEventTracker.kt

```diff
@@ -396,26 +455,25 @@ class TraceCollectingEventTracker(
 
     override fun afterWrite() = Unit
 
-    override fun afterLocalRead(codeLocation: Int, variableId: Int, value: Any?) = runInsideIgnoredSection {
-        val threadData = ThreadDescriptor.getCurrentThreadDescriptor()?.eventTrackerData as? ThreadData? ?: return
-
-        // TODO: make in optional
-/*
-        val tracePoint = TRReadLocalVariableTracePoint(
-            threadId = threadData.threadId,
-            codeLocationId = codeLocation,
-            localVariableId = variableId,
-            value = TRObjectOrNull(value)
-        )
-        strategy.tracePointCreated(threadData.currentMethodCallTracePoint(), tracePoint)
-*/
+    override fun afterLocalRead(codeLocation: Int, variableId: Int, value: Any?) = runInsideInjectedCode {
+        val threadDescriptor = ThreadDescriptor.getCurrentThreadDescriptor() ?: return
+        val threadData = threadDescriptor.eventTrackerData as? ThreadData? ?: return
 
+            // TODO: make in optional
+//        val tracePoint = TRReadLocalVariableTracePoint(
+//            threadId = threadData.threadId,
+//            codeLocationId = codeLocation,
+//            localVariableId = variableId,
+//            value = TRObjectOrNull(value)
+//        )
+//        strategy.tracePointCreated(threadData.currentMethodCallTracePoint(), tracePoint)
```

Comment:
please fix indentation here

Reply:
indentation for the commented-out code is correct. Fixed the indentation for the TODO above